<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Programmende</class>
 <widget class="QDialog" name="Dialog_Programmende">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programmende</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>419</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_amNullpunkt">
          <property name="text">
           <string>am Nullpunkt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_benutzerdefiniert">
          <property name="text">
           <string>an Benutzerdefinierter Position</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="2">
         <widget class="QLineEdit" name="lineEdit_ausfuehrbedingung"/>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lineEdit_pos_z"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_pos_y">
          <property name="text">
           <string>Position in Y:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEdit_pos_x"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEdit_pos_y"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_Ausfuerbedingung">
          <property name="text">
           <string>Ausf√ºhrbedingung</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_pos_z">
          <property name="text">
           <string>Position in Z:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_pos_x">
          <property name="text">
           <string>Position in X:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>[Y]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>[X]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>[Z]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>[AFB]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="lineEdit_bezeichnung"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label__bezeichnung_var">
          <property name="text">
           <string>[BEZ]</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_bezeichnung">
          <property name="text">
           <string>Bezeichnung:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_abbrechen">
        <property name="text">
         <string>Abbrechen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="text">
         <string>Speichern</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_OK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>radioButton_amNullpunkt</tabstop>
  <tabstop>radioButton_benutzerdefiniert</tabstop>
  <tabstop>lineEdit_pos_x</tabstop>
  <tabstop>lineEdit_pos_y</tabstop>
  <tabstop>lineEdit_pos_z</tabstop>
  <tabstop>lineEdit_bezeichnung</tabstop>
  <tabstop>lineEdit_ausfuehrbedingung</tabstop>
  <tabstop>pushButton_abbrechen</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_OK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
